src/Agent/NewRelic/Agent/Core/ThreadProfiling/ThreadProfilingService.cs

```diff
@@ -101,6 +103,7 @@ public void Start()
         /// Stops the <see cref="ThreadProfilingService"/> service. This will halt a 
         /// thread profiling session that might be running.
         /// </summary>
+        [NrExcludeFromCodeCoverage]
```

Comment:
It seems like you could potentially write a meaningful test that goes through the lifecycle using the excluded methods. The test could simply call Start, StartSession, SampleAcquired, Stop, SamplingComplete, and then verify that we got the expected result. A test like that might more closely mimic the interaction of a thread profiling run.

Reply:
@nrcventura I re-worked one of my tests to do a full-cycle and removed the exclude attributes. Thanks for the suggestion!